tests/ha/test_ha_node_health.py

```diff
@@ -171,80 +160,60 @@ def test_nodes_one_by_one_os_shutdown(self):
             assert_utils.assert_true(resp[0], resp[1])
             sys_obj = resp[1]
             check_rem_node = [
-                "failed" if num == node else "online" for num in range(
-                    self.num_nodes)]
-            resp = self.ha_obj.verify_node_health_status(
-                sys_obj, status=check_rem_node)
+                "failed" if num == node else "online" for num in range(self.num_nodes)]
+            resp = self.ha_obj.verify_node_health_status(sys_obj, status=check_rem_node)
             assert_utils.assert_true(resp[0], resp[1])
             resp = self.ha_rest.verify_node_health_status_rest(check_rem_node)
             assert_utils.assert_true(resp[0], resp[1])
 
             LOGGER.info("Check for the node down alert.")
-            resp = self.csm_alerts_obj.verify_csm_response(
-                self.starttime, self.alert_type["get"], False, "iem")
+            resp = self.csm_alerts_obj.verify_csm_response(self.starttime, self.alert_type["get"],
+                                                           False, "iem")
             assert_utils.assert_true(resp, "Failed to get alert in CSM")
             # TODO: If CSM REST getting changed, add alert check from msg bus
 
-            LOGGER.info(
-                "Check that cortx services on other nodes are not affected.")
-            resp = self.ha_obj.check_service_other_nodes(
-                node, self.num_nodes, self.node_list)
-            assert_utils.assert_true(
-                resp, "Some services are down for other nodes.")
+            LOGGER.info("Check that cortx services on other nodes are not affected.")
+            resp = self.ha_obj.check_service_other_nodes(node, self.num_nodes, self.node_list)
+            assert_utils.assert_true(resp, "Some services are down for other nodes.")
 
-            LOGGER.info("Power on {}".format(node_name))
             resp = self.ha_obj.host_power_on(host=self.host_list[node], bmc_obj=self.bmc_list[node])
-            assert_utils.assert_true(
-                resp, "Host has not powered on yet.")
-            LOGGER.info("{} has powered on".format(node_name))
+            assert_utils.assert_true(resp, "Host has not powered on yet.")
             self.restored = True
             # To get all the services up and running
             time.sleep(40)
             if self.setup_type == "HW":
-                LOGGER.debug(
-                    "HW: Need to enable stonith on the node after node came back up")
-                self.node_list[node].execute_cmd(
-                    common_cmds.PCS_RESOURCE_STONITH_CMD.format("enable", node+1),
-                    read_lines=True)
+                LOGGER.debug("HW: Need to enable stonith on the node after node came back up")
+                self.node_list[node].execute_cmd(common_cmds.PCS_RESOURCE_STONITH_CMD.format(
+                    "enable", node+1), read_lines=True)
```

Comment:
same node + 1, plz check wherever if this chnage is applicable

Reply:
> same node + 1, plz check wherever if this chnage is applicable

May be @RajeshPratapSInghChouhan can suggest.

Reply:
same as above

Reply:
Spaces around +